livemaps-api/APIFunctions/PostState.cs

```diff
@@ -86,7 +84,7 @@ public static class PostState
                             value = jp.Value.ToString();
                         }
                     }
-                                        
+
                     var stateSetCfg = buildingConfig.StateSets.FirstOrDefault(ss => ss.StateSetName.ToLower() == stateSet.ToLower());
 
                     if(stateSetCfg != null)
```

Comment:
The brace on L93 should be on a new line

Reply:
@jsquire I have updated the format, please review.